src/dbos-executor.ts

```diff
@@ -106,6 +107,9 @@ import {
 export interface DBOSNull {}
 export const dbosNull: DBOSNull = {};
 
+export const DBOS_QUEUE_MIN_PRIORITY = 0;
```

Comment:
This should be `1` to be consistent with Python. By default we have 0, but user should only specify priority from 1~ 2,147,483,647